api/resources/translations/messages-fr.properties

```
@@ -373,6 +373,7 @@ app.version.unknown =
 associated.contact = Contact associé
 associated.contact.help = Lorsque cet utilisateur crée des rapport, ils seront assignés à ce contact
 autoreply = auto-réponse
+barcode_scanner.error.cannot_read_barcode = Impossible de lire le code-barres. Retenter.
```

@m5r is the French okay? 😬

---

```suggestion
barcode_scanner.error.cannot_read_barcode = Impossible de lire le code barre. Veuillez réessayer.
```

Almost! I would phrase it this way instead

---

Thanks!